rules/windows/process_creation/win_susp_vboxdrvInst.yml

```diff
@@ -0,0 +1,30 @@
+title: Suspicious VBoxDrvInst.exe Parameters
+id: b7b19cb6-9b32-4fc4-a108-73f19acfe262
+description: Detect VBoxDrvInst.exe run whith parameters allowing processing INF file. This allows to create values in the registry and install drivers.
+    For example one could use this technique to obtain persistence via modifying one of Run or RunOnce registry keys
+status: experimental
+author: Konstantin Grishchenko, oscd.community
+date: 2020/10/06
+references:
+    - https://github.com/LOLBAS-Project/LOLBAS/blob/master/yml/LOLUtilz/OtherBinaries/VBoxDrvInst.yml
+    - https://twitter.com/pabraeken/status/993497996179492864
+tags:
+    - attack.defense_evasion
+    - attack.t1112
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        Image|endswith: '\VBoxDrvInst.exe'
+        CommandLine:
+            - 'driver*executeinf'
```

Comment:
Hello @grikos !

Thank you for your contribution! 

The rule looks great, I would only change this line to the following:

```
        CommandLine|contains|all:
            - 'driver'
            - 'executeinf'
```

Some LM/SIEM systems don't process '`*`' as Splunk or Elasticsearch, so it won't work out of the box for them, and it would be hard for a SIEM expert to understand how exactly he/she supposes to construct a correlation rule looking at that detection logic.

Reply:
Hello @grikos! Could you please fix that?

Reply:
@yugoslavskiy fixed
![image](https://user-images.githubusercontent.com/51186173/95908238-734b1f00-0da5-11eb-8826-bcd7940350a8.png)

Reply:
Thanks, @grikos! Well done!